### DISCUSSION TITLE
Planning to make a big code refactoring

### DISCUSSION THREAD

**Discussion by Nevysha** [Nevysha/Cozy-Nest#33] [Announcements]
Just so you know, I'm planning to make a big code refactoring because the JS file start to be to big and It's difficult to nav inside it. I'll break it into multiple JS file and the script.js entry point will take charge of loading each file.

**Comment by swordworksAI** [comment]
Browser (Edge) Full Screen Bug Feedback: When I press F11 to make the browser full screen, Cozy's UI does not change. I think this should be a bug and hope to fix it as soon as possible.
